README.md

```diff
@@ -182,6 +197,36 @@ $googleAdsClient = (new GoogleAdsClientBuilder())
 Once you have an instance of `GoogleAdsClient`, you can obtain a service client
 for a particular service using one of the `get...ServiceClient()` methods.
 
+## Protobuf
+
+[Protobuf](https://developers.google.com/protocol-buffers/docs/overview) is used by
+[gRPC](https://grpc.io/about/), a core dependency of the Google Ads API for request and
+response transport.
+
+There are two implementations available for PHP
+
+1. C (native): for better performance
+1. PHP (pure): to get started easily
+
+By default implementation of Protobuf used is C. This might not be suitable for some users
```

Comment:
Nit: how about "By default, the C implementation is used"?

Reply:
Done